### PR TITLE
Update dive log screens and defaults

### PR DESCRIPTION
**Pull request by mjmayank**
Edits to profile page:
image

### CODE REVIEW

File: src/Screens/Logs/Log.tsx [hunk 1]
```diff
@@ -161,6 +167,17 @@ const Log: FunctionComponent<LogProps> = ({ navigation, route }) => {
       });
     };
 
+    const deleteDiveLog = () => {
+      const url = `${config.API_ENDPOINT}/review/delete?review_id=${diveLog.review.id}`;
+      fetch(url, {
+        method: 'POST',
+        headers: {
```
Comment (Nnanyielugo) [inline src/Screens/Logs/Log.tsx:174]: I think we should move this function to `redux/slices/dive-logs/api`

**Review by Nnanyielugo — APPROVED** [review]
Looks good, apart from the delete dive log function that should be moved to the file where similar functions live.

File: src/Screens/Logs/Log.tsx (hunk 1 above)
Comment (mjmayank) [reply src/Screens/Logs/Log.tsx]: fixed

File: src/Screens/DiveLogsForms/AdvancedDiveLogsForm.tsx [hunk 2]
```diff
@@ -244,28 +244,22 @@ const AdvancedDiveLogsForm: FunctionComponent<AdvancedDiveLogsFormsProps> = ({
...
-      : user?.unit === 'imperial'
-      ? 57.2
-      : 14,
+      : undefined,
     air_temp: simpleDiveLogsForm.air_temp
       ? simpleDiveLogsForm.air_temp
-      : user?.unit === 'imperial'
-      ? 60
-      : 20,
+      : undefined,
```
[the remark this answers is not in the document]
Comment (mjmayank) [reply src/Screens/DiveLogsForms/AdvancedDiveLogsForm.tsx:256]: yeah, i don't want users to accidentally submit incorrect values. i'd rather have it be undefined

**Pull request merged**